scripts/fix-revision.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
Can we use something like this https://stackoverflow.com/questions/42518067/how-to-use-ordereddict-as-an-input-in-yaml-dump-or-yaml-safe-dump to keep the ordering. Having changeset before name seems odd.